packages/design-system/src/components/Review/ReviewLink.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useCallback } from 'react';
+
+export interface ReviewLinkProps {
+  /**
+   * Provide this value to give screenreaders longer, more descriptive text to
+   * explain the context of the link.
+   */
+  ariaLabel?: string;
+  className?: string;
+  children: React.ReactNode;
+  href: string;
+  onClick?: (...args: any[]) => any;
+}
+
+export const ReviewLink = (props: ReviewLinkProps) => {
+  const handleClick = useCallback(
+    (event) => {
+      if (props.onClick) {
+        props.onClick(event, props.href);
+      }
+    },
+    [props.onClick, props.href]
+  );
```

Comment:
I think `useCallback` [is overkill](https://kentcdodds.com/blog/usememo-and-usecallback#why-is-usecallback-worse) here because it's unlikely that those two prop dependencies will change after first render.

Reply:
good to know!